lib/pretty.js

```diff
@@ -147,11 +147,13 @@ function pretty (inputData) {
       this.messageKey,
       this.levelKey,
       this.timestampKey
-    ].filter(key => {
-      return typeof log[key] === 'string' ||
-        typeof log[key] === 'number' ||
-        typeof log[key] === 'boolean'
-    })
+    ]
+      .map((key) => key.replaceAll(/\\/g, ''))
```

Comment:
This strips off the `\\` in `log\\.level` so that you get the correct key to ignore.